### DISCUSSION TITLE
SMTP bug in new release

### DISCUSSION THREAD

**Discussion by rgraham7503** [hyperjumptech/monika#750] [General]
I just updated Monika/NPM on MAC OS and Windows and now smtp notification is broken. I am showing proof here of it working, then after the update (which you can see) now its not working?? Seems like a BUG in the new release.
`PS C:\Users\Administrator\monika-node> monika -c .\auth_monitor.yml
INFO: Monika is running from: Dallas - xxxxx (xxxxx) - xxxxx (xxxxxx)
INFO: Updating config
Using config file: C:\Users\Administrator\monika-node\auth_monitor.yml
Starting Monika. Probes: 1. Notifications: 2`

`INFO: 2022-07-05T14:13:57.076Z - Connected to STUN Server. Monika is running from: xxxxxx
INFO: 2022-07-05T14:18:09.946Z - Connected to STUN Server. Monika is running from: xxxxxx
INFO: 2022-07-05T14:18:10.574Z 1 id:isv_auth 200 POST https://xxxxx.com/v1.0/endpoint/default/token 597ms
INFO: 2022-07-05T14:18:11.500Z 1 id:isv_auth 200 POST https://xxxxx.com/v1.0/authnmethods/password/xxxxxx?returnJwt=true
899ms
INFO: 2022-07-05T14:18:29.947Z - Connected to STUN Server. Monika is running from: xxxxx
INFO: Thank you for using Monika!
INFO: We need your help to make Monika better.
INFO: Can you give us some feedback by clicking this link https://github.com/hyperjumptech/monika/discussions?`

`77 packages are looking for funding`
 `run npm fund for details`
 `4 vulnerabilities (1 moderate, 3 high)`

`To address all issues (including breaking changes), run:`
  `npm audit fix --force`

`Run npm audit for details.`
`npm notice`
`npm notice New minor version of npm available! 8.5.5 -> 8.13.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.13.2
npm notice Run npm install -g npm@8.13.2 to update!
npm notice
PS C:\Users\Administrator\monika-node>
PS C:\Users\Administrator\monika-node> monika -c .\auth_monitor.yml
INFO: Monika is running from: Dallas - xxxxxx (xxxxxx) - xxxxxx (xxxxxx)
INFO: Updating config
C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\node_modules\@oclif\core\lib\errors\index.js:29
        err = new cli_2.CLIError(input, options);`
        
`CLIError: Failed to send message using SMTP, please check your SMTP notification config.`
`Message: C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\node_modules\mailgen/themes/default/index.html:396`
 `394|             </td>`
`395|           </tr>`
`>> 396|           <% if (locals.action && locals.action[0].button[0].fallback) { %>`
`397|             <tr>`
`398|               <td class="email-body" width="100%">`
`399|                 <table class="email-body_inner" align="center" width="570" cellpadding="0" cellspacing="0">`
`Cannot read properties of undefined (reading 'button')
at Object.error (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\node_modules\@oclif\core\lib\errors\index.js:29:15)
at Monika.error (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\node_modules\@oclif\core\lib\command.js:90:23)
at Monika.run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\lib\index.js:191:18)
at async Monika._run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\@hyperjumptech\monika\node_modules\@oclif\core\lib\command.js:67:22) {
oclif: { exit: 1 },
code: undefined
}`

**Comment by sapiderman** [comment]
@rgraham7503 hi thanks for the feedback!

Looking into it now

**Reply by sapiderman** [reply]
Confirmed issue on SMTP on Windows AND Mac. Strangely no problems on Linux.

**Comment by sapiderman** [comment]
Created a ticket: #763
